client/components/jetpack/scan-threats-new/index.tsx

```diff
@@ -0,0 +1,347 @@
+import { Button, Card } from '@automattic/components';
+import { numberFormat, translate } from 'i18n-calypso';
+import * as React from 'react';
+import { useDispatch } from 'react-redux';
+import FoldableCard from 'calypso/components/foldable-card';
+import FixAllThreatsDialog from 'calypso/components/jetpack/fix-all-threats-dialog-new';
+import SecurityIcon from 'calypso/components/jetpack/security-icon';
+import ThreatDialog from 'calypso/components/jetpack/threat-dialog-new';
+import ThreatItem from 'calypso/components/jetpack/threat-item-new';
+import {
+	FixableThreat,
+	Threat,
+	ThreatAction,
+} from 'calypso/components/jetpack/threat-item-new/types';
+import ThreatListHeader from 'calypso/components/jetpack/threat-list-header';
+import ThreatLowRiskItemHeader from 'calypso/components/jetpack/threat-low-risk-item-header';
+import SupportInfo from 'calypso/components/support-info';
+import contactSupportUrl from 'calypso/lib/jetpack/contact-support-url';
+import { triggerScanRun } from 'calypso/lib/jetpack/trigger-scan-run';
+import { useThreats } from 'calypso/lib/jetpack/use-threats';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+import './style.scss';
+
+interface Site {
+	ID: number;
+	name: string;
+	URL: string;
+}
+
+interface Props {
+	site: Site;
+	threats: Array< Threat >;
+	error: boolean;
+}
+
+const ScanError: React.FC< { site: Site } > = ( { site } ) => {
+	const dispatch = useDispatch();
+	const dispatchScanRun = React.useCallback( () => {
+		triggerScanRun( site.ID )( dispatch );
+	}, [ dispatch, site ] );
+
+	return (
+		<div className="scan-threats-new__error">
+			{ translate(
+				'The scanner was unable to check all files and errored before completion. Deal with the threats found above and run the {{runScan}}scan again{{/runScan}}. If the error persists, we are {{linkToSupport}}here to help{{/linkToSupport}}.',
+				{
+					components: {
+						runScan: (
+							<Button className="scan-threats-new__run-scan-button" onClick={ dispatchScanRun } />
+						),
+						linkToSupport: (
+							<a href={ contactSupportUrl( site.URL ) } rel="noopener noreferrer" target="_blank" />
+						),
+					},
+				}
+			) }
+		</div>
+	);
+};
+
+const getThreatCountMessage = (
+	countHighSeverity: number,
+	countLowSeverity: number,
+	prefix: string | null,
+	suffix: string | null
+) => {
+	let lowThreatsSummary = '';
+	if ( countLowSeverity ) {
+		lowThreatsSummary = translate( '%(lowCount)s low risk item', '%(lowCount)s low risk items', {
+			args: {
+				lowCount: numberFormat( countLowSeverity, 0 ),
+			},
+			comment: '$(lowCount)s is the number of low severity items found.',
+			count: countLowSeverity,
+		} );
+	}
+
+	let highThreatsSummary = '';
+	if ( countHighSeverity ) {
+		highThreatsSummary = translate( '%(threatCount)s threat', '%(threatCount)s threats', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '%(threatCount)d threat found', '%(threatCount)d threats found' )` *ES Score: 7*